src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -89,6 +89,6 @@ public function getEntitiesByIds($identifier, array $values)
         return $qb->andWhere($where)
                   ->getQuery()
                   ->setParameter($parameter, $values, $parameterType)
-                  ->getResult();
+                  ->getResult() ?: [];
```

Comment:
this change looks wrong to me. getResult returns an array (unless you change the hydration mode of the query, but this is not the case here, and would also fail for successful results too anyway)

Reply:
agreed, fixed in 0a00af2b77